### DISCUSSION TITLE
Error importing a design with Spaceclaim

### DISCUSSION THREAD

**Discussion by IFarshidI** [ansys/pyansys-geometry#1282] [Q&A] [answered]
Hello,

I am encountering an error when trying to import a design. I am using Ansys 24R1 version and pyAnsys Geometry version 0.6.0 is installed. I have imported launch modeler with spaceclaim and tried to open a file and got an error indicating that the file apparently is larger than the maximum size. 
 
-The error looks like this: 
GeometryExitedError: Geometry service connection terminated: Received message larger than max (280828563 vs. 268435456)

I'd really appreciate it if you could let me know how the issue can be resolved.

**Comment by RobPasMue** [comment, marked as answer]
Hi @IFarshidI! 

Could you share the PyAnsys Geometry script you are using so that I try to replicate it on my side? Also, would it be possible to share the file with us?

I have the feeling that you might be trying to "upload" the file to SpaceClaim, which is the default behavior of ``open_file()`` (see https://geometry.docs.pyansys.com/version/stable/api/ansys/geometry/core/modeler/Modeler.html#Modeler.open_file)

Could you give it a try to ``design = modeler.open_file(..., upload_to_server=False)``?

**Reply by IFarshidI** [reply]
yes, that was the issue. I didn't specify whether to upload the file or not. Thanks a lot for the quick response! :)

**Reply by RobPasMue** [reply]
Awesome! Thank you for using PyAnsys Geometry!
